core/src/main/java/nu/yona/server/analysis/entities/GoalConflictMessage.java

```diff
@@ -125,22 +119,17 @@ public static GoalConflictMessage createInstanceFromBuddy(UUID relatedUserAnonym
 	{
 		if (origin == null)
 		{
-			throw GoalServiceException.originCannotBeNull();
+			throw new IllegalArgumentException("origin cannot be null");
```

Comment:
Let's stick to the pattern that Phillip introduced and use specific and localizable exceptions everywhere.